www/js/diary/infinite_scroll_place_item.js

```diff
@@ -0,0 +1,39 @@
+
+/**
+ * A directive to display each trip within the diary view.
+ */
+
+angular.module('emission.main.diary.infscrollplaceitem', [
+                                                        'emission.main.diary.infscrolllist',
+                                                        'emission.survey.multilabel.services',
+                                                        'emission.main.diary.infscrolldetail',
+                                                        'ng-walkthrough',
+                                                        'nvd3', 'emission.plugin.kvstore',
+                                                        'emission.services',
+                                                        'emission.config.imperial',
+                                                        'emission.plugin.logger',
+                                                        'emission.stats.clientstats',
+                                                        'emission.survey.enketo.time-use',
+])
+
+.directive("infiniteScrollPlaceItem", function(){
+    return{
+      restrict: 'E',
+      scope: {
+        trip: '='
+      },
+      controller: 'PlaceItemCtrl',
+      templateUrl: 'templates/diary/place_list_item.html'
+    };
+  })
+
+  .controller("PlaceItemCtrl", function($scope, $state) {
+    console.log("Place Item Controller called");
+
+    // TODO detail page for places
+    // $scope.showDetail = function($event) {
+    //   $state.go("root.main.inf_scroll-detail", {
+    //  
+    //   });
+    // }
```

Comment:
I don't think we need a detail page for places; the plan right now is to have a list of place based user inputs in the item. Note that this implies that the item needs to resize when we add new place based user inputs.

Reply:
I was going to ask about whether a detail page is necessary. Thanks for clearing that up 👍

`infinite_scroll_list` uses `collection-repeat`, which relies on each item having a fixed height. To have dynamic heights, we will need a different scrolling solution.

I can easily just swap in `ng-repeat`, but I believe it is less performant. Best solution would be to tap into native scrolling. But can we even do that in this version of ionic?

Reply:
we came up with a solution to resize collection-repeat elements - we needed it for the replaced mode display. Please check with @sebastianbarry on details.